runXcodeTests.sh

```diff
@@ -11,15 +11,8 @@ if [ -z "$1" ]
 fi
 
 xctool \
```

Comment:
Definitely possible. We don't use it internally and if it's not hooked up to CI here, it's bound to bit rot.